README.md

```diff
@@ -7,6 +7,14 @@ EvilPlot is about combinators for graphics.
 
 Documentation for EvilPlot is available [here](https://cibotech.github.com/evilplot).
 
+## Getting Started
+Add EvilPlot to your build with:
+```scala
+resolvers += Resolver.bintrayRepo("cibotech", "public")
+libraryDependencies += "com.cibo" %% "evilplot" % "0.2.0" // Use %%% instead of %% if you're using ScalaJS
```

Comment:
current version is 0.2.1.
I guess we could also just make a note that the current version is displayed at the top on the badge. Or update this on releases.